### PR TITLE
Added support for TheWiz Hebrew subtitles website.

### PR DESCRIPTION
**Pull request by ofir123**
Fixes #

Proposed changes in this pull request:
- Added support for TheWiz - A new Hebrew subtitles website.
- While a similar PR is open for Subliminal, I opened it here as well to speed things up.
- Thanks :)

- [ ] PR is based on the DEVELOP branch
- [ ] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [ ] Read [contribution guide](https://github.com/SickRage/SickRage/blob/master/.github/CONTRIBUTING.md)

### CODE REVIEW

**Comment by natiz** [comment]
+1

**Comment by miigotu** [comment]
This can be added to SR without modifying the lib/subliminal, which is a vanilla source copy of subliminal.
Look at how we add itasa provider in `SickRage/sickrage/providers/subtitles/itasa.py` and then register it in `SickRage/sickbeard/subtitles.py`

**Comment by ofir123** [comment]
@miigotu Done! It now works externally, same as itasa.
Thnx for the tip :)

**Comment by IpDo** [comment]
+1

File: lib/subliminal/extensions.py [hunk 1]
```diff
@@ -95,6 +95,7 @@ def unregister(self, entry_point):
     'shooter = subliminal.providers.shooter:ShooterProvider',
     'subscenter = subliminal.providers.subscenter:SubsCenterProvider',
     'thesubdb = subliminal.providers.thesubdb:TheSubDBProvider',
+    'thewiz = subliminal.providers.thewiz:TheWizProvider',
```
Comment (miigotu) [inline lib/subliminal/extensions.py]: this should not be here

File: lib/subliminal/providers/thewiz.py [hunk 2]
```diff
@@ -0,0 +1,198 @@
+# -*- coding: utf-8 -*-
```
Comment (miigotu) [inline lib/subliminal/providers/thewiz.py]: this file should not be here

**Comment by ofir123** [comment]
Fixed and merged with develop branch again.

**Review by sharkykh — APPROVED** [review]

**Pull request merged**